elements/pfe-accordion/src/pfe-accordion.story.js

```diff
@@ -0,0 +1,93 @@
+import { storiesOf } from "@storybook/polymer";
+import * as storybookBridge from "@storybook/addon-knobs/polymer";
+import * as tools from "../../../.storybook/utils.js";
+
+const cleaner = require("clean-html");
+
+import PfeAccordion from "../pfe-accordion";
+
+const stories = storiesOf("Accordion", module);
+
+const defaultHeader = tools.autoHeading();
+const defaultContent = tools.autoContent(5, 3);
+
+// Define the template to be used
+const template = (data = {}) => {
+  return tools.component(PfeAccordion.tag, data.prop, data.slots);
+};
+
+stories.addDecorator(storybookBridge.withKnobs);
+
+stories.add(PfeAccordion.tag, () => {
+  let config = {};
+  // const props = PfeAccordion.properties;
+  // const slots = PfeAccordion.slots;
+
+  //-- Add content to light DOM
+  config.slots = [];
+
+  // Let the user determine number of accordions
+  let accordionCount = storybookBridge.number("Count", 5, {
+    min: 1,
+    max: 10
+  });
+
+  // Let the user customize the first header + panel set
+  let header = storybookBridge.text("Header", defaultHeader, "accordion-set");
+  let content = storybookBridge.text("Panel", defaultContent, "accordion-set");
+
+  config.slots.push({
+    content:
+      tools.component("pfe-accordion-header", {}, [
+        {
+          content: tools.customTag({
+            tag: "h2",
+            content: header
+          })
+        }
+      ]) +
+      tools.component("pfe-accordion-panel", {}, [
+        {
+          content: content
+        }
+      ])
+  });
+
+  // Use dynamic content for the rest
+  for (let i = 1; i < accordionCount; i++) {
+    config.slots.push({
+      content:
+        tools.component("pfe-accordion-header", {}, [
+          {
+            content: tools.customTag({
+              tag: "h2",
+              content: tools.autoHeading()
+            })
+          }
+        ]) +
+        tools.component("pfe-accordion-panel", {}, [
+          {
+            content: tools.autoContent(5, 3)
+          }
+        ])
+    });
+  }
+
+  let rendered = template(config);
+
+  cleaner.clean(
+    rendered,
+    {
+      indent: "    ",
+      "remove-attributes": [],
+      wrap: 0
+    },
+    html => (rendered = html)
+  );
+
+  return `${rendered}
+<pre style="white-space: pre-wrap; padding: 20px 50px; background-color: #f0f0f0; font-weight: bold;border: 1px solid #bccc;">
```

Comment:
@castastrophe - Would we want to add an additional stylesheet to the preview pane head instead of doing inline styles?

https://storybook.js.org/configurations/add-custom-head-tags/

Reply:
It might be excessive to create a stylesheet just for this short set of styles.  How about I create a single-source of truth instead via a preview function in the utils that each story would call?

Reply:
bc9262f1e732cfa23cd8f6f82129f6a475b62d41